scribble-lib/scribble/acmart.rkt

```diff
@@ -9,7 +10,31 @@
          (for-syntax racket/base))
 
 (provide/contract
- [title (->* (pre-content?) (#:short pre-content?) content?)]
+ [title (->* (pre-content?)
+             (#:short pre-content?
+              #:tag (or/c string? (listof string?) #f)
+              #:tag-prefix (or/c string? module-path? #f)
+              #:style (or/c style? string? symbol? #f)
+              #:version (or/c string? #f)
+              #:date (or/c string? #f))
+             title-decl?)]
+ [author (->* () () #:rest (listof pre-content?)
```

Comment:
Is there a good reason to keep both `author` and `authorinfo` in the API? Seems like `authorinfo` is the "right one".